src/ksm/ksm_test.go

```diff
@@ -0,0 +1,63 @@
+package ksm_test
+
+// This file holds the integration tests for the KSM package.
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/newrelic/nri-kubernetes/v2/internal/config"
+	"github.com/newrelic/nri-kubernetes/v2/internal/testutil"
+	"github.com/newrelic/nri-kubernetes/v2/src/ksm"
+	ksmClient "github.com/newrelic/nri-kubernetes/v2/src/ksm/client"
+	"github.com/newrelic/nri-kubernetes/v2/src/metric"
+)
+
+func TestScraper(t *testing.T) {
+	// TODO(roobre): We should not exclude Optional, pod or hpa metrics. To be tackled in a follow-up PR.
+	asserter := testutil.NewAsserter().
+		Using(metric.KSMSpecs).
+		ExcludingOptional().
+		Excluding("pod").
+		Excluding("hpa")
```

Comment:
we are excluding them since in the output we have locally those metrics were not present?

Reply:
Yes, this is temporary. The static files don't have those metrics. In an upcoming PR this, together with the `ExcludingOptional` will be removed and the static files updated.